build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -0,0 +1,157 @@
+                            RestoreDesktopTestAssetProjects;
+                            BuildTestAssetProjects;
+                            BuildDesktopTestAssetProjects;" />
+
+  <Target Name="SetupTestPackages" 
+          DependsOnTargets="RestoreTestAssetPackages;
+                            SetupTestPackageProjectData"
+          Outputs="%(TestPackageProject.Identity)">
+    <PropertyGroup>
+      <VersionSuffixOption></VersionSuffixOption>
+      <VersionSuffixOption Condition="'%(TestPackageProject.VersionSuffix)' != ''">--version-suffix %(TestPackageProject.VersionSuffix)</VersionSuffixOption>
+    </PropertyGroup>
+    <Exec Command="$(DotnetUnderTest) pack %(TestPackageProject.FullPath) --output $(TestPackagesDir) $(VersionSuffixOption)" />
+  </Target>
+
+  <Target Name="RestoreTestAssetPackages" 
+          DependsOnTargets="PrepareTests;
+                            SetupRestoreTestAssetPackagesInputs;" 
+          Inputs="@(RestoreTestAssetPackagesInputs)"
+          Outputs="@(RestoreTestAssetPackagesInputs->'%(RelativeDir)/project.lock.json')">
+    <Exec Command="$(DotnetUnderTest) restore $(RepoRoot)/TestAssets/TestPackages/" />
+  </Target>
+
+  <Target Name="SetupRestoreTestAssetPackagesInputs">
+    <ItemGroup>
+      <RestoreTestAssetPackagesInputs Include="$(RepoRoot)/TestAssets/TestPackages/**/project.json" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="RestoreTestAssetProjects"
+          DependsOnTargets="SetupTestPackages;
+                            SetupRestoreTestAssetProjectsInputs;"
+          Inputs="@(RestoreTestAssetProjectsInputs)"
+          Outputs="@(RestoreTestAssetProjectsInputs->'%(RelativeDir)/project.lock.json')">
+    <Exec Command="$(DotnetUnderTest) restore $(RepoRoot)/TestAssets/TestProjects/ --fallbacksource $(TestPackagesDir)" />
+  </Target>
+
+  <Target Name="SetupRestoreTestAssetProjectsInputs" >
+    <ItemGroup>
+      <RestoreTestAssetProjectsInputs Include="$(RepoRoot)/TestAssets/TestProjects/**/project.json" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CleanTestAssetProjectsLockFiles" >
+    <ItemGroup>
+      <TestAssetProjectsLockFiles Include="$(RepoRoot)/TestAssets/TestProjects/**/project.lock.json" />
+    </ItemGroup>
+    <Delete Files="@(TestAssetProjectsLockFiles)" /> 
+  </Target>
+
+  <Target Name="RestoreDesktopTestAssetProjects"
+          DependsOnTargets="SetupTestPackages;
+                            SetupRestoreDesktopTestAssetProjectsInputs;"
+          Inputs="@(RestoreDesktopTestAssetProjectsInputs)"
+          Outputs="@(RestoreDesktopTestAssetProjectsInputs->'%(RelativeDir)/project.lock.json')">
+    <Exec Command="$(DotnetUnderTest) restore $(RepoRoot)/TestAssets/DesktopTestProjects/ --fallbacksource $(TestPackagesDir)" />
+  </Target>
+
+  <Target Name="SetupRestoreDesktopTestAssetProjectsInputs">
+    <ItemGroup>
+      <RestoreDesktopTestAssetProjectsInputs Include="$(RepoRoot)/TestAssets/TestProjects/**/project.json" />
```

Comment:
How is this different than the <RestoreTestAssetProjectsInputs> item?  It looks the same, shouldn't it point to the "Desktop" test asset projects?